politeiad/config.go

```diff
@@ -66,14 +73,15 @@ type config struct {
 	DebugLevel  string   `short:"d" long:"debuglevel" description:"Logging level for all subsystems {trace, debug, info, warn, error, critical} -- You may also specify <subsystem>=<level>,<subsystem2>=<level>,... to set the log level for individual subsystems -- Use show to list available subsystems"`
 	Listeners   []string `long:"listen" description:"Add an interface/port to listen for connections (default all interfaces port: 49152, testnet: 59152)"`
 	Version     string
-	HTTPSCert   string `long:"httpscert" description:"File containing the https certificate file"`
-	HTTPSKey    string `long:"httpskey" description:"File containing the https certificate key"`
-	RPCUser     string `long:"rpcuser" description:"RPC user name for privileged commands"`
-	RPCPass     string `long:"rpcpass" description:"RPC password for privileged commands"`
-	DcrtimeHost string `long:"dcrtimehost" description:"Dcrtime ip:port"`
-	DcrtimeCert string `long:"dcrtimecert" description:"File containing the https certificate file for dcrtimehost"`
-	Identity    string `long:"identity" description:"File containing the politeiad identity file"`
-	GitTrace    bool   `long:"gittrace" description:"Enable git tracing in logs"`
+	HTTPSCert   string   `long:"httpscert" description:"File containing the https certificate file"`
+	HTTPSKey    string   `long:"httpskey" description:"File containing the https certificate key"`
+	RPCUser     string   `long:"rpcuser" description:"RPC user name for privileged commands"`
+	RPCPass     string   `long:"rpcpass" description:"RPC password for privileged commands"`
+	DcrtimeHost string   `long:"dcrtimehost" description:"Dcrtime ip:port"`
+	DcrtimeCert string   `long:"dcrtimecert" description:"File containing the https certificate file for dcrtimehost"`
+	Identity    string   `long:"identity" description:"File containing the politeiad identity file"`
+	GitTrace    bool     `long:"gittrace" description:"Enable git tracing in logs"`
+	MimeTypes   []string `long:"mimetypes" description:"Valid mimetypes accepted by politeiad"`
```

Comment:
Why are you using a slice here if the only use the first element?

Reply:
I used a slice because the default mime types we set are described in a slice; once we parse the string it also returns a slice. Should that change in any way?